From 416e5fdbb11ae83f739c214db9c7dbb021b89269 Mon Sep 17 00:00:00 2001
From: Steve Sakoman <steve@sakoman.com>
Date: Thu, 3 Mar 2011 13:29:30 -0800
Subject: [PATCH 01/30] Revert "omap2_mcspi: Flush posted writes"

This reverts commit a330ce2001b290c59fe98c37e981683ef0a75fdf.

The above commit breaks the ads7846 driver
---
 drivers/spi/spi-omap2-mcspi.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/spi/spi-omap2-mcspi.c b/drivers/spi/spi-omap2-mcspi.c
index 0c73dd4..d9848fe 100644
--- a/drivers/spi/spi-omap2-mcspi.c
+++ b/drivers/spi/spi-omap2-mcspi.c
@@ -189,7 +189,6 @@ static inline void mcspi_write_chconf0(const struct spi_device *spi, u32 val)
 
 	cs->chconf0 = val;
 	mcspi_write_cs_reg(spi, OMAP2_MCSPI_CHCONF0, val);
-	mcspi_read_cs_reg(spi, OMAP2_MCSPI_CHCONF0);
 }
 
 static void omap2_mcspi_set_dma_req(const struct spi_device *spi,
-- 
1.7.9.5

